docs/nut-command/database/database-export.md

```diff
@@ -24,25 +17,36 @@ file.
 
 ## Options
 
-| Option | Description |
-|--------|-------------|
-| -c, --contenttypes=CONTENTTYPES | One or more ContentTypes to export records for. (multiple values allowed)
-| -f, --file=FILE | A YAML or JSON file to use for export data. Must end with .yml, .yaml or .json
-
+| Option                          | Description |
+|---------------------------------|-------------|
+| `-f, --file=FILE`               | A YAML or JSON file to use for export data. Must end with .yml, .yaml or .json
+| `-d, --directory=DIRECTORY`     | A destination directory. The command will automatically generate file names.
+| `-c, --contenttype=CONTENTTYPE` | ContentType name to export records for (can be used multiple times). (multiple values allowed)
+| `-h, --help `                   | Display this help message
+| `-q, --quiet `                  | Do not output any message
+| `-V, --version`                 | Display this application version
+| `    --ansi`                    | Force ANSI output
+| `    --no-ansi`                 | Disable ANSI output
+| `-n, --no-interaction`          | Do not ask any interactive question
+| `-v|vv|vvv, --verbose`          | Increase the verbosity of messages: 1 for normal output, 2 for more verbose output and 3 for debug
```

Comment:
The "help" though "verbose" options are just part of Symfony Console, and we should not document them here.

Reply:
Updated.. I left in the '-n', though, because it's particularly handy if you want to use this scripted. :-)

Reply:
Until people start complaining about inconsistency across the docs and trying to add them left and right :face_with_head_bandage:

Reply:
My plan was to go over all of the nut commands docs. Quite a few others need updates too. We can decide that I leave in (or add) the `-n` for all of them, where relephant?